process/using_copykit_artifacts/README.md

```diff
@@ -0,0 +1,121 @@
+# Getting Prototype Pages Into App
+
+This is a Guide to using the files/artefacts created by copykit
+
+For details of copykit itself see
+
+* CopyKit repo : <https://github.com/EnvironmentAgency/copykit>
```

Comment:
Please switch to using **-** instead of * for unordered lists. Trying to settle on a convention and have gone for this as it is more clearly differentiated. * is prevalent when it comes to styling text e.g. **bold**, *italicised*, ***both*** are often done using *.